### ISSUE TITLE
Segmentation Fault with MLT v7.0.0

### ISSUE THREAD

**Issue by chromer030** [mltframework/shotcut#1086]
I installed latest shotcut throught Arch Linux repository but with any operation like New , add video , add audio etc , crash and segmentation fault , as you can see in log below :

Shotcut 21,05,01
MLT 7
Arch Linux
Linux : 5.11.16
Desktop : KDE Plasma 5.21.4

```$ shotcut         
[Info   ] <Application::Application> Starting Shotcut version ARCH-21.05.01 
[Info   ] <Application::Application> Linux version 
[Info   ] <Application::Application> number of logical cores = 2 
[Info   ] <Application::Application> locale = QLocale(English, Latin, United States) 
[Info   ] <Application::Application> install dir = "/usr/bin" 
[Debug  ] <ShotcutSettings::log> language "en_US" 
[Debug  ] <ShotcutSettings::log> deinterlacer "onefield" 
[Debug  ] <ShotcutSettings::log> external monitor "" 
[Debug  ] <ShotcutSettings::log> GPU processing false 
[Debug  ] <ShotcutSettings::log> interpolation "bilinear" 
[Debug  ] <ShotcutSettings::log> video mode "atsc_1080p_30" 
[Debug  ] <ShotcutSettings::log> realtime true 
[Debug  ] <ShotcutSettings::log> audio channels 2 
[Debug  ] <MainWindow::changeTheme> begin 
[Debug  ] <MainWindow::MainWindow> begin 
[Info   ] <MainWindow::MainWindow> device pixel ratio = 1 
[Info   ] <GLTestWidget::GLTestWidget> OpenGL context version 3 0 
[Debug  ] <Mlt::Controller::Controller> begin 
[Error  ] <MLT> mlt_repository_init: no plugins found in "/build/mlt/src/build/out/lib/mlt"
[Info   ] <Mlt::Controller::resetLocale> decimal point . 
[Debug  ] <Mlt::Controller::Controller> end 
[Debug  ] <Mlt::GLWidget::GLWidget> begin 
[Debug  ] <Mlt::GLWidget::GLWidget> end 
[Debug  ] <MainWindow::setupSettingsMenu> begin 
[Debug  ] <MainWindow::setupSettingsMenu> end 
[Debug  ] <MainWindow::readPlayerSettings> begin 
[Debug  ] <MainWindow::setAudioChannels> 2 
[Debug  ] <Mlt::Controller::setAudioChannels> 2 
[Debug  ] <MainWindow::setPreviewScale> 0 
[Debug  ] <Mlt::Controller::setPreviewScale> 720 x 576 
[Debug  ] <MainWindow::readPlayerSettings> end 
[Debug  ] <MainWindow::configureVideoWidget> begin 
[Debug  ] <MainWindow::setProfile> "atsc_1080p_30" 
[Debug  ] <Mlt::Controller::setProfile> setting to profile "atsc_1080p_30" 
[Debug  ] <MainWindow::setAudioChannels> 2 
[Debug  ] <Mlt::Controller::setAudioChannels> 2 
[Debug  ] <MainWindow::configureVideoWidget> end 
[Debug  ] <ScopeController::ScopeController> begin 
[Debug  ] <ScopeWidget::ScopeWidget> begin true 
[Debug  ] <ScopeWidget::ScopeWidget> end 
[Debug  ] <AudioLoudnessScopeWidget::AudioLoudnessScopeWidget> begin 
[Debug  ] <AudioLoudnessScopeWidget::AudioLoudnessScopeWidget> end 
[Debug  ] <ScopeDock::ScopeDock> begin 
[Debug  ] <ScopeDock::ScopeDock> end 
[Debug  ] <ScopeWidget::ScopeWidget> begin true 
[Debug  ] <ScopeWidget::ScopeWidget> end 
[Debug  ] <AudioPeakMeterScopeWidget::AudioPeakMeterScopeWidget> begin 
[Debug  ] <AudioPeakMeterScopeWidget::AudioPeakMeterScopeWidget> end 
[Debug  ] <ScopeDock::ScopeDock> begin 
[Debug  ] <ScopeDock::ScopeDock> end 
[Debug  ] <ScopeWidget::ScopeWidget> begin true 
[Debug  ] <ScopeWidget::ScopeWidget> end 
[Debug  ] <AudioSpectrumScopeWidget::AudioSpectrumScopeWidget> begin 
[Debug  ] <AudioSpectrumScopeWidget::AudioSpectrumScopeWidget> end 
[Debug  ] <ScopeDock::ScopeDock> begin 
[Debug  ] <ScopeDock::ScopeDock> end 
[Debug  ] <ScopeWidget::ScopeWidget> begin true 
[Debug  ] <ScopeWidget::ScopeWidget> end 
[Debug  ] <AudioWaveformScopeWidget::AudioWaveformScopeWidget> begin 
[Debug  ] <AudioWaveformScopeWidget::AudioWaveformScopeWidget> end 
[Debug  ] <ScopeDock::ScopeDock> begin 
[Debug  ] <ScopeDock::ScopeDock> end 
[Debug  ] <ScopeWidget::ScopeWidget> begin true 
[Debug  ] <ScopeWidget::ScopeWidget> end 
[Debug  ] <VideoHistogramScopeWidget::VideoHistogramScopeWidget> begin 
[Debug  ] <VideoHistogramScopeWidget::VideoHistogramScopeWidget> end 
[Debug  ] <ScopeDock::ScopeDock> begin 
[Debug  ] <ScopeDock::ScopeDock> end 
[Debug  ] <ScopeWidget::ScopeWidget> begin true 
[Debug  ] <ScopeWidget::ScopeWidget> end 
[Debug  ] <VideoRgbParadeScopeWidget::VideoRgbParadeScopeWidget> begin 
[Debug  ] <VideoRgbParadeScopeWidget::VideoRgbParadeScopeWidget> end 
[Debug  ] <ScopeDock::ScopeDock> begin 
[Debug  ] <ScopeDock::ScopeDock> end 
[Debug  ] <ScopeWidget::ScopeWidget> begin true 
[Debug  ] <ScopeWidget::ScopeWidget> end 
[Debug  ] <VideoRgbWaveformScopeWidget::VideoRgbWaveformScopeWidget> begin 
[Debug  ] <VideoRgbWaveformScopeWidget::VideoRgbWaveformScopeWidget> end 
[Debug  ] <ScopeDock::ScopeDock> begin 
[Debug  ] <ScopeDock::ScopeDock> end 
[Debug  ] <ScopeWidget::ScopeWidget> begin true 
[Debug  ] <ScopeWidget::ScopeWidget> end 
[Debug  ] <VideoVectorScopeWidget::VideoVectorScopeWidget> begin 
[Debug  ] <VideoVectorScopeWidget::profileChanged> 601 
[Debug  ] <VideoVectorScopeWidget::VideoVectorScopeWidget> end 
[Debug  ] <ScopeDock::ScopeDock> begin 
[Debug  ] <ScopeDock::ScopeDock> end 
[Debug  ] <ScopeWidget::ScopeWidget> begin true 
[Debug  ] <ScopeWidget::ScopeWidget> end 
[Debug  ] <VideoWaveformScopeWidget::VideoWaveformScopeWidget> begin 
[Debug  ] <VideoWaveformScopeWidget::VideoWaveformScopeWidget> end 
[Debug  ] <ScopeDock::ScopeDock> begin 
[Debug  ] <ScopeDock::ScopeDock> end 
[Debug  ] <ScopeWidget::ScopeWidget> begin true 
[Debug  ] <ScopeWidget::ScopeWidget> end 
[Debug  ] <VideoZoomWidget::VideoZoomWidget> begin 
[Debug  ] <VideoZoomWidget::VideoZoomWidget> end 
[Debug  ] <VideoZoomScopeWidget::VideoZoomScopeWidget> begin 
[Debug  ] <VideoZoomScopeWidget::VideoZoomScopeWidget> end 
[Debug  ] <ScopeDock::ScopeDock> begin 
[Debug  ] <ScopeDock::ScopeDock> end 
[Debug  ] <ScopeController::ScopeController> end 
[Debug  ] <RecentDock::RecentDock> begin 
[Debug  ] <RecentDock::RecentDock> end 
[Debug  ] <PlaylistDock::PlaylistDock> begin 
[Debug  ] <PlaylistDock::PlaylistDock> end 
[Debug  ] <TimelineDock::TimelineDock> begin 
[Debug  ] <TimelineDock::TimelineDock> end 
[Debug  ] <FiltersDock::FiltersDock> begin 
[Debug  ] <FiltersDock::FiltersDock> end 
[Debug  ] <KeyframesDock::KeyframesDock> begin 
[Debug  ] <KeyframesDock::KeyframesDock> end 
[Debug  ] <EncodeDock::EncodeDock> begin 
[Debug  ] <EncodeDock::EncodeDock> end 
[Debug  ] <JobsDock::JobsDock> begin 
[Debug  ] <JobsDock::JobsDock> end 
[Debug  ] <MainWindow::readWindowSettings> begin 
[Debug  ] <MainWindow::readWindowSettings> end 
[Debug  ] <LeapNetworkListener::start> begin 
[Debug  ] <MainWindow::MainWindow> end 
[Debug  ] <MainWindow::changeTheme> end 
[Debug  ] <AudioLoudnessScopeWidget::resetQview> begin 
[Debug  ] <FiltersDock::resetQview> begin 
[Debug  ] <KeyframesDock::load> begin 
[Debug  ] <Mlt::GLWidget::initializeGL> begin 
[Info   ] <Mlt::GLWidget::initializeGL> OpenGL vendor "X.Org" 
[Info   ] <Mlt::GLWidget::initializeGL> OpenGL renderer "AMD RV730 (DRM 2.50.0 / 5.11.16-arch1-1, LLVM 11.1.0)" 
[Info   ] <Mlt::GLWidget::initializeGL> OpenGL threaded? true 
[Info   ] <Mlt::GLWidget::initializeGL> OpenGL ES? false 
[Info   ] <Mlt::GLWidget::initializeGL> OpenGL maximum texture size = 8192 
[Info   ] <Mlt::GLWidget::initializeGL> OpenGL maximum viewport size = 8192 x 8192 
[Debug  ] <Mlt::GLWidget::initializeGL> end 
[Debug  ] <NewProjectFolder::showEvent> m_profile "atsc_1080p_30" action.data "atsc_1080p_30" 
[Info   ] <Util::isMemoryLow> available RAM =  3708640 KB 
[Debug  ] <Worker::run> Database version is 1 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "alpha_adjust" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "alpha_view" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "audio_balance" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "audio_bandpass" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "audio_basstreble" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "audio_channelcopy" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "audio_compressor" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "audio_delay" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "audio_expander" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "audio_fadein" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "audio_fadeout" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "audio_gain" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "audio_highpass" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "audio_invert" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "audio_limiter" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "audio_lowpass" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "audio_mono" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "audio_mute" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "audio_noisegate" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "audio_normalize_1p" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "audio_normalize_2p" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "audio_notch" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "audio_pan" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "audio_pitch" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "audio_reverb" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "audio_swapchannels" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "bigsh0t_eq_mask" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "bigsh0t_eq_to_rect" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "bigsh0t_hemi_to_eq" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "bigsh0t_rect_to_eq" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "bigsh0t_stabilize_360" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "bigsh0t_transform_360" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "blend_mode" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "bluescreen0r" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "blur" "meta_boxblur.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "blur" "meta_movit.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "blur_exponential" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "blur_gaussian" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "blur_lowpass" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "brightness" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "brightness" "meta_movit.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "choppy" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "chromahold" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "color" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "color" "meta_frei0r_coloradj.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "color" "meta_movit.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "contrast" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "contrast" "meta_movit.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "corners" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "crop" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "crop" "meta_movit.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "crop_circle" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "crop_rectangle" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "dance" "meta_dance.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "distort" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "dither" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "dust" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "dynamictext" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "elastic_scale" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "fadein_brightness" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "fadein_movit" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "fadeout_brightness" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "fadeout_movit" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "flip" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "flip" "meta_movit.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "fspp" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "glitch" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "glow" "meta_frei0r.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "glow" "meta_movit.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "gradient" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "grain" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "grid" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "halftone" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "hqdn3d" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "hue_lightness_saturation" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "invert" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "keyspillm0pup" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "lenscorrection" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "levels" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "lightshow" "meta_lightshow.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "lines" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "lut3d" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "mask" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "mask_alphaspot" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "mask_apply" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "mask_shape" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "mirror" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "mirror" "meta_movit.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "mosaic" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "movit_diffusion" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "nervous" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "noise_fast" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "noise_keyframes" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "nosync" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "oldfilm" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "opacity" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "opacity" "meta_movit.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "pillar_echo" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "posterize" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "rgbsplit0r" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "richtext" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "rotate" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "saturation" "meta_frei0r.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "saturation" "meta_movit.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "scanlines" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "select0r" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "sepia" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "sharpen" "meta_frei0r.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "sharpen" "meta_movit.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "size_position" "meta_affine.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "size_position" "meta_movit.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "sketch" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "smartblur" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "spectrum" "meta_spectrum.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "spillsuppress" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "spot_remover" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "stabilize" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "tcolor" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "threshold" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "time_remap" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "timer" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "trails" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "unpremultiply" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "vaguedenoiser" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "vertigo" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "vignette" "meta_movit.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "vignette" "meta_oldfilm.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "wave" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "waveform" "meta.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "white" "meta_frei0r.qml" 
[Debug  ] <FilterController::loadFilterMetadata> reading filter metadata "white" "meta_movit.qml" 
[Debug  ] <LeapNetworkListener::onDisconnected> Disconnected from Leap Motion 
[Debug  ] <LeapNetworkListener::onError> Leap Motion WebSocket error: "Connection refused" 
[Debug  ] <MainWindow::open> "/home/chromer/.local/share/Meltytech/Shotcut/__untitled__.mlt" 
[Debug  ] <MltXmlChecker::check> begin 
[Debug  ] <MltXmlChecker::check> "" 
[Debug  ] <MltXmlChecker::check> end "" 
[Info   ] <MainWindow::open> decimal point '.' 
[Debug  ] <Timeline::AddTrackCommand::redo> audio 
zsh: segmentation fault (core dumped)  shotcut```

**Comment by ddennedy** [comment]
Sorry but we do not start and debug other parties’ builds. Please try with one of ours.

**Comment by chromer030** [comment]
This bug is reported in Arch Linux Bug Tracker several times :

https://bugs.archlinux.org/task/69442
https://bugs.archlinux.org/task/69274
https://bugs.archlinux.org/task/70695

I dont know if the issue is in side of Arch Linux or upstream.
Feel free to close this issue it culprit is Arch.
Thank you.

**Comment by ddennedy** [comment]
I think I found a build-related problem in the new MLT7 when it is used for a system install:
https://github.com/mltframework/mlt/issues/645#issuecomment-831556662

I am testing it now and probably going to make a MLT 7.0.1 version soon. The other bugs you linked except the last are not related as this problem is specific to these new versions.

**Comment by chromer030** [comment]
I tried Appimage and no any crash and problem , seg fault is fault of Arch in build and packaging.

**Comment by ddennedy** [comment]
@eszlari Are you waiting on the fix I mentioned above to update the flatpak? It is in git, and I will release 7.0.1 with it very soon.

**Comment by chromer030** [comment]
After posting some backtrace here : https://bugs.archlinux.org/task/70695 , Arch maintainers fixed the issue about this crash, it was a problem in MLT building , fixed in latest official package on Arch repo , i think this issue can be closed.
Thank you all.

**Comment by hfiguiere** [comment]
> @eszlari Are you waiting on the fix I mentioned above to update the flatpak? It is in git, and I will release 7.0.1 with it very soon.

I was trying to update the flatpak and I did hit that bug.

**Comment by ddennedy** [comment]
Closing this as resolved due to the bug fix made for MLT v7.0.1 https://github.com/mltframework/mlt/commit/d0ff2b5a6586ddc0a0509c016ae56bdb80741034
